package.json

```diff
@@ -13,18 +13,6 @@
     "dist",
     "app-router"
   ],
-  "exports": {
```

Comment:
Turns out we need to add this back in to be imported correctly in the nextjs pages project

```json
  "exports": {
    "./app-router": {
      "import": "./dist/app-router/handlers/enable-draft.js",
      "require": "./dist/app-router/handlers/enable-draft.js",
      "types": "./dist/app-router/handlers/enable-draft.d.ts"
    },
    "./pages-router": {
      "import": "./dist/pages-router/handlers/enable-draft.js",
      "require": "./dist/pages-router/handlers/enable-draft.js",
      "types": "./dist/pages-router/handlers/enable-draft.d.ts"
    }
  },
```